src/components/shared/Validations.tsx

```diff
@@ -54,19 +54,19 @@ const createValidationsFormatter = (name: string, options?: { exact?: boolean; n
   const values = Array.isArray(value) ? value : [value];
   if (values.length) {
     return {
-      name: options?.exact ? name : values.length > 1 ? `${name} values` : `${name} value`,
+      name: options?.exact ? name : values.length > 1 ? `${name}s` : `${name}`,
       values: values.map(createStringFormatter(options?.nowrap)),
     };
   }
   return null;
 };
 
 const validationFormatters: Record<string, (value: unknown) => ValidationFormat | null> = {
-  enum: createValidationsFormatter('Allowed', { nowrap: true }),
+  enum: createValidationsFormatter('Allowed value', { nowrap: true }),
```

Comment:
```suggestion
  enum: createValidationsFormatter('Allowed values', { nowrap: true }),
```
shouldn't this say "values"?

Reply:
The `createValidationsFormatter` pluralizes it if needed